utils/build/docker/java/parametric/src/main/java/com/datadoghq/trace/opentelemetry/controller/OpenTelemetryController.java

```diff
@@ -58,11 +44,13 @@ public class OpenTelemetryController {
   private final Tracer tracer;
   private final TextMapPropagator propagator;
   private final Map<Long, Span> spans;
+  private Baggage baggage;
```

Comment:
Does this survive across multiple requests to the same controller? In .NET a new controller instance is created for each request, so these are static members.

https://github.com/DataDog/system-tests/blob/9c26730e12231319fb71aa0572c45d6a01676b1a/utils/build/docker/dotnet/parametric/Endpoints/ApmTestApi.cs#L73-L74